### PR TITLE
add dizzy animation

### PR DESCRIPTION
**Pull request by iraqwarvet31**
Currently I was looking for this type of animation on this site, but it didn't exist. Here is the demo:
https://codepen.io/mELTingAwaY__/pen/bGqdpGx?editors=1111

As you can see my animation works on codepen however, will not work locally on the site animate.css. When I click on the list item "dizzy" the animation isn't working. The 2 classes are being applied as you can see in the attached screenshot. Also to note the 2 classes aren't being removed like the other animations. Is there another file that is required to change or are there issues with my animation?

### CODE REVIEW

**Comment by WarenGonzaga** [comment]
I'll review your code and @eltonmesquita can you please take a look into this?

**Comment by WarenGonzaga** [comment]
Done, reviewing! I like it and I can confirm that this is not existing animation in our library. I might tweak it a bit.
Let's add this to v5 @eltonmesquita!

**Review by warengonzaga — APPROVED** [review]

File: animate.css [hunk 1]
```diff
@@ -863,6 +863,35 @@
...
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+
+  to {
+    -webkit-transform: rotate(1440deg);
+    transform:rotate(1440deg);
+  }
+}
+@keyframes dizzy {
```
Comment (syedabdullahali) [inline animate.css:878]: why u did not used % percentage method of @keyframes

**Review by syedabdullahali — APPROVED** [review]

**Comment by eltonmesquita** [comment]
Sorry for the massive delay. I prefer to skip it for now, but I might get back to it in the future. Thanks for the contribution!

**Pull request closed without merging**